script/update

```diff
@@ -299,6 +312,46 @@ def download(url, filename):
           break
         f.write(chunk)
 
+def get_packaged_src(clean, git_cache, nohooks, upload_src):
+  src_version = chromium_version()
+  tar_name = 'src.tar.bz2'
+  package_url = '{0}/{1}/src/{2}/{3}'.format(CHROMIUMCONTENT_SRC_URL,
+                                         PLATFORM_KEY, src_version, tar_name)
+  try:
+    print 'Trying to download ' + package_url
+    filesystem.download_and_extract(SOURCE_ROOT, package_url)
+  except urllib2.URLError as err:
+    print('URLError while downloading {0}, syncing from git: {1}'.format(package_url, err.reason))
+    gclient_sync(src_version, clean, git_cache, nohooks)
+    if upload_src:
+      print 'Packaging source to ' + tar_name + '.'
+      tar_path =  os.path.join(SOURCE_ROOT, tar_name)
+      filesystem.safe_unlink(tar_path)
+      with util.scoped_cwd(SOURCE_ROOT):
+        tar_file = tarfile.open(tar_path, 'w:bz2')
+        tar_file.add('src', exclude=exclude_packaged_files)
```

Comment:
i think adding `.gclient` and `.gclient_entries` here will satisfy gclient's search for the root directory and obviate the need for `CHROMIUM_BUILDTOOLS_PATH`. Maybe. 😬